### ISSUE TITLE
Improve performance to handle large number of repos/workflows

### ISSUE THREAD

**Issue by kishaningithub** [otto-de/gitactionboard#913]
**Is your feature request related to a problem? Please describe.**
Would like to use this to monitor pipelines for a large github org with ~100 repos having several workflows. I find that it takes several minutes (10-15 mins) for initial load and the performance is incredibly slow.

**Describe the solution you'd like**
Increase the performance, keeping in mind the github api limits to have dashboard for ~100 repos with several workflows

**Describe alternatives you've considered**
No alternatives

**Additional context**
Instead of rest api i suggest doing an experiment with the graph ql api so that we can avoid n*n calls to improve the performance.

**Comment by kishaningithub** [comment]
Looks like graph api has some serious limitations to achieve what this tool strives to do
https://github.com/orgs/community/discussions/24493

**Comment by sumanmaity112** [comment]
You are right @kishaningithub, for this tool to properly work we need to make M*N REST API call. Currently I am working on a feature to have secondary caching using SQLite, theoretically it'll reduce the load time for consecutive restarts.
For the first time it'll take longer but eventually down the line it'll reduce the overall load time

**Comment by sumanmaity112** [comment]
@kishaningithub, given v5.0.x is already released with SQLite as secondary cache, I am thinking to close this issue. What's your opinion?

**Comment by kishaningithub** [comment]
Yes we can 🎉☺️👍

**Comment by sumanmaity112** [comment]
This issue is resolved as part [V5.0.0](https://github.com/otto-de/gitactionboard/discussions/979#discussion-7942545).
